yarpl/test/FlowableTest.cpp

```diff
@@ -593,6 +586,22 @@ TEST(FlowableTest, SubscribeMultipleTimes) {
   EXPECT_EQ(results[4], std::vector<int64_t>({1, 2, 3, 4, 5}));
 }
 
+/* following test should probably behave like:
```

Comment:
?

Reply:
We discussed how exceptions should bubble up in the meeting, sounds like we need to discuss it further though. I don't think it makes sense to swallow exceptions at the Subscriber level; that's rsocket's job. I left this note here for a future design discussion about it.

Reply:
why do we need commented out code. can you remove it?